src/flat-file-adapter.js

```diff
@@ -0,0 +1,36 @@
+const flatfile = require('flat-file-db')
+const promise = require('promise')
+const dateFns = require('date-fns')
+
+const db = flatfile.sync(process.env.DB_NAME || 'views.db')
+
+module.exports = {
+  put: promise.denodeify(db.put.bind(db)),
+
+  has: (key) => Promise.resolve(db.has(key)),
+  get: (key, options) => {
+    const value = db.get(key) || { views: [] }
+
+    return {
+      views: value.views.filter(view => {
+        if (options && options.before && dateFns.isAfter(view.time, options.before)) return false
+        if (options && options.after && dateFns.isBefore(view.time, options.after)) return false
```

Comment:
well the thought is that before and after are to be integer values(UTC millies) atm. Which, with that assumption, it's not the adapter's job to make it so. I think index.js is the place that needs to handle this:

```
data: await db.getAll({
           pathname: pathname,
           before: query.before && parseInt(query.before),
           after: query.after && parseInt(query.after),
         }),
```
or something to that effect